system-test/fixtures/google-gax-packaging-test-app/.prettierrc

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
why are the two prettier configs in this PR different?

Reply:
My bad! Fixed.